examples/training/train_eraser.py

```diff
@@ -0,0 +1,550 @@
+                            },
+                            {"mean": 0.0, "std": 1.0},
+                        ],
+                    )
+                ),
+                RescaleMapper(RescaleMapperConfig(key="before", verbose=True)),
+                RescaleMapper(RescaleMapperConfig(key="after", verbose=True)),
+            ],
+        ),
+    ]
+
+    return filters_mappers
+
+
+def get_data_module(
+    train_shards: List[str],
+    validation_shards: List[str],
+    batch_size: int,
+):
+
+    # TRAIN
+    train_filters_mappers = get_filter_mappers()
+
+    # unbrace urls
+    train_shards_path_or_urls_unbraced = []
+    for train_shards_path_or_url in train_shards:
+        train_shards_path_or_urls_unbraced.extend(
+            braceexpand.braceexpand(train_shards_path_or_url)
+        )
+
+    # shuffle shards
+    random.shuffle(train_shards_path_or_urls_unbraced)
+
+    # data config
+    data_config = DataModuleConfig(
+        shards_path_or_urls=train_shards_path_or_urls_unbraced,
+        decoder="pil",
+        shuffle_before_split_by_node_buffer_size=20,
+        shuffle_before_split_by_workers_buffer_size=20,
+        shuffle_before_filter_mappers_buffer_size=20,
+        shuffle_after_filter_mappers_buffer_size=20,
+        per_worker_batch_size=batch_size,
+        num_workers=min(10, len(train_shards_path_or_urls_unbraced)),
+    )
+
+    train_data_config = data_config
+
+    # VALIDATION
+    validation_filters_mappers = get_filter_mappers()
+
+    # unbrace urls
+    validation_shards_path_or_urls_unbraced = []
+    for validation_shards_path_or_url in validation_shards:
+        validation_shards_path_or_urls_unbraced.extend(
+            braceexpand.braceexpand(validation_shards_path_or_url)
+        )
+
+    data_config = DataModuleConfig(
+        shards_path_or_urls=validation_shards_path_or_urls_unbraced,
+        decoder="pil",
+        shuffle_before_split_by_node_buffer_size=10,
+        shuffle_before_split_by_workers_buffer_size=10,
+        shuffle_before_filter_mappers_buffer_size=10,
+        shuffle_after_filter_mappers_buffer_size=10,
+        per_worker_batch_size=batch_size,
+        num_workers=min(10, len(train_shards_path_or_urls_unbraced)),
+    )
+
+    validation_data_config = data_config
+
+    # data module
+    data_module = DataModule(
+        train_config=train_data_config,
+        train_filters_mappers=train_filters_mappers,
+        eval_config=validation_data_config,
+        eval_filters_mappers=validation_filters_mappers,
+    )
+
+    return data_module
+
+
+def main(
+    train_shards: List[str] = ["pipe:cat path/to/train/shards"],
+    validation_shards: List[str] = ["pipe:cat path/to/validation/shards"],
+    backbone_signature: str = "stable-diffusion-v1-5/stable-diffusion-v1-5",
+    vae_num_channels: int = 4,
+    unet_input_channels: int = 4,
+    source_key: str = "before",
+    target_key: str = "after",
+    mask_key: str = "mask",
+    neptune_project: str = "LBM Eraser",
```

Comment:
nit: you've used "LBM-Eraser" in `eraser.yaml` (maybe keep the same default here)